openc3-cosmos-init/plugins/packages/openc3-vue-common/src/plugins/plugin-store/PluginStore.vue

```diff
@@ -0,0 +1,153 @@
+          Browse plugins made by OpenC3 and the COSMOS community
+        </div>
+      </div>
+      <v-spacer />
+      <v-btn icon="mdi-cog" variant="text" @click="openSettings" />
+
+      <v-tooltip location="top">
+        <template #activator="{ props }">
+          <div v-bind="props">
+            <v-btn
+              href="https://store.openc3.com"
+              target="_blank"
+              icon="mdi-open-in-new"
+              variant="text"
+              class="mx-2"
+            />
+          </div>
+        </template>
+        <span> Open store.openc3.com </span>
+      </v-tooltip>
+
+      <v-btn icon="mdi-close" variant="text" @click="close" />
+    </div>
+    <div class="d-inline-flex align-center w-100">
+      <v-text-field
+        v-model="search"
+        label="Search"
+        prepend-inner-icon="mdi-magnify"
+        clearable
+        variant="outlined"
+        density="compact"
+        single-line
+        hide-details
+        data-test="search-plugin-store"
+      />
+      <v-switch
+        v-model="verifiedOnly"
+        label="Verified only"
+        density="compact"
+        hide-details
+        class="ml-4"
+      />
+    </div>
+    <v-spacer />
+    <span class=""> Click on a plugin to see more information about it </span>
+    <v-container>
+      <v-row>
+        <v-col v-for="plugin in filteredPlugins" :key="plugin.id" cols="4">
+          <plugin-card v-bind="plugin" @trigger-install="install" />
+        </v-col>
+      </v-row>
+    </v-container>
+  </v-sheet>
+  <plugin-store-settings-dialog
+    v-model="showSettingsDialog"
+    @update:store-url="updatePluginStore"
+  />
+</template>
+
+<script>
+import { Api, OpenC3Api } from '@openc3/js-common/services'
+import { PluginCard } from '@/plugins/plugin-store'
+import PluginStoreSettingsDialog from './PluginStoreSettingsDialog.vue' // idk why importing from @/plugins/plugin-store isn't working
+
+export default {
+  components: {
+    PluginCard,
+    PluginStoreSettingsDialog,
+  },
+  emits: ['close', 'triggerInstall'],
+  data() {
+    return {
+      api: new OpenC3Api(),
+      search: '',
+      verifiedOnly: false,
+      showSettingsDialog: false,
+      plugins: [],
+    }
+  },
+  computed: {
+    filteredPlugins: function () {
+      let filtered = this.plugins
+      if (this.verifiedOnly) {
+        filtered = filtered.filter((plugin) => plugin.verified)
+      }
+      if (this.search.length) {
+        filtered = filtered.filter((plugin) =>
+          plugin.title.toLowerCase().includes(this.search.toLowerCase()),
+        )
+      }
+      return filtered
+    },
+  },
+  mounted: function () {
+    // Don't call updatePluginStore() here. It should be called in the background before the user opens this store
+    // view (see PluginsTab.vue `created()`) to keep this feeling fast
+    this.fetchPluginStoreData()
+    // TODO: do something with plugins that are already installed (show uninstall button instead?)
```

Comment:
No uninstall button.  Still just install because some plugins you want to install multiple time (multiple power supplies)